gr-uhd/swig/uhd_swig.i

```diff
@@ -117,10 +117,40 @@
 
 %include <uhd/stream.hpp>
 
+%include <uhd/types/filters.hpp>
+
+%include stdint.i
+
+// Used for lists of filter taps
+%template(uhd_vector_int16_t) std::vector<int16_t>;
+
+%template(filter_info_base_sptr) boost::shared_ptr<uhd::filter_info_base>;
```

Comment:
Dang it -- these are std::shared_ptr in UHD 4.0.

Reply:
LOL Progress! Same issue with different Thrift versions ... at least UHD provides version macros that are reliable and easily usable!

Reply:
I'll wait to review until you get the `shard_ptr` thingy fixed up ;)

Reply:
Martin added a UHD version check that addresses this.